### DISCUSSION TITLE
Typography valid HTML tags

### DISCUSSION THREAD

**Discussion by Mil4n0r** [dxc-technology/halstack-react#1842] [General] [answered]
Related to issue #1621.

The subset of allowed tags for the Typography component has to be decided.

Proposal:

`"a" | "blockquote" | "cite" | "code" | "div" | "em" | "figcaption" | "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "p" | "pre" | "small" | "span" | "strong"`

**Comment by kukoc** [comment]
I agree with that list. I would maybe include lists as well:  <ul>, <ol>, <li> (unordered list, ordered list, list items)

**Reply by Mil4n0r** [reply]
Is that not already covered by the `BulletedList` component?

**Reply by kukoc** [reply]
Agree. Sometimes I forget about this component

**Comment by Mil4n0r** [comment, marked as answer]
For now, I think we can go on with the selection that I proposed. 

If we detect any use cases that are not covered, we can always update it in the future 😃 

#1765
